### DISCUSSION TITLE
Add attributes to get method of a expression-bodied property

### DISCUSSION THREAD

**Discussion by acaly** [dotnet/csharplang#5014] [Language Ideas] [answered]
For example, can we do this?

```c#
[method: MethodImpl(MethodImplOptions.AggressiveInlining)]
public int Count => GetTheCount();
```

instead of

```c#
public int Count
{
    [MethodImpl(MethodImplOptions.AggressiveInlining)]
    get => GetTheCount();
}
```

One may argue that the inliner can decide quite good on whether to inline, but actually besides the aggressive lining option, there are plenty of usages of this pattern even in .NET repo, for example, the `ReadOnlySpan<T>.Length` is:

```c#
public int Length
{
    [NonVersionable]
    get => _length;
}
```

which could be
```c#
[method: NonVersionable]
public int Length => _length;
```

**Comment by svick** [comment, marked as answer]
A very similar proposal: https://github.com/dotnet/csharplang/issues/4425.
